Implementation of getguildinfo script command (#8935)
rathena/rathena: doc/script_commands.txt

---------------------------------------

*getguildinfo(<guild ID>, <type>);

This function will look up and return the request type of guild information.

Types:
	GUILDINFO_NAME              - Guild's Name

> 	GUILDINFO_LEVEL,            - Guild's Level
> 	GUILDINFO_AVERAGELEVEL,     - Guild's Average Level
> 	GUILDINFO_ONLINECOUNT,      - Guild's Online Member Count
> 	GUILDINFO_MEMBERCOUNT,      - Guild's Current Member Count
> 	GUILDINFO_MAXMEMBERCOUNT,   - Guild's Max Member Count
> 	GUILDINFO_EXP,              - Guild's Current EXP
> 	GUILDINFO_NEXTEXP,          - Guild's Required EXP to Level
> 	GUILDINFO_MASTERID,         - Guild Master Character ID
> 	GUILDINFO_MASTERNAME,       - Guild Master Name

Reviewer: ```suggestion
	GUILDINFO_LEVEL             - Guild's Level
	GUILDINFO_AVERAGELEVEL      - Guild's Average Level
	GUILDINFO_ONLINECOUNT       - Guild's Online Member Count
	GUILDINFO_MEMBERCOUNT       - Guild's Current Member Count
	GUILDINFO_MAXMEMBERCOUNT    - Guild's Max Member Count
	GUILDINFO_EXP               - Guild's Current EXP
	GUILDINFO_NEXTEXP           - Guild's Required EXP to Level
	GUILDINFO_MASTERID          - Guild Master Character ID
	GUILDINFO_MASTERNAME        - Guild Master Name
```

> Note: Make sure to use *requestguildinfo to sync the latest guild data from the char-server

Reviewer: ```suggestion
Note: Make sure to use the requestguildinfo script command to load the guild data from the char-server.
```

Example Case ( Newly formed guild ):

getguildinfo( getcharid(2), GUILDINFO_LEVEL ); //Returns 0

> requestguildinfo( getcharid(2) );
> getguildinfo( getcharid(2), GUILDINFO_LEVEL ); //Returns current guild level

Reviewer: ```suggestion
.@gid = 1234;
requestguildinfo( .@gid );
getguildinfo( .@gid, GUILDINFO_LEVEL ); //Returns current guild level
```
Using getcharid means the guild will always be loaded already.

---------------------------------------

*getguildinfo(<guild ID>, <type>);

This function will look up and return the request type of guild information.

> The function will return -1 on failure.

Reviewer 2: The command can return a string (for example with GUILDINFO_NAME) or int (GUILDINFO_LEVEL), 
I suggest that the command stops the script on failure since there is no data type defined

```suggestion
```
